sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/variant/variantExpressions.scala

```diff
@@ -73,6 +73,45 @@ case class ParseJson(child: Expression)
     copy(child = newChild)
 }
 
+@ExpressionDescription(
+  usage = "_FUNC_(expr) - Check if a variant value is a variant NULL.",
```

Comment:
We'd better be explicit in the doc: it returns true if and only if the input is a variant null (conventionally, we use lower case to refer to a variant null). For all other cases, including the input is a SQL NULL, we return false.

Reply:
Resolved